webapp/tests/settings.py

```diff
@@ -51,3 +51,5 @@ def atexit_tmpremover(dirname):
 INDEX_FILE = os.path.join(TEMP_GRAPHITE_DIR, 'index')
 
 URL_PREFIX = '/graphite'
+
+CARBON_CACHE_FINDER = 'tests.fixtures.EmptyFinder'
```

Comment:
doing this fixes some unit tests that were failing with the carbon cache finder.